### PR TITLE
improve trailer

### PR DESCRIPTION
**Pull request by TomGallon**

### CODE REVIEW

File: app/views/movie/detail/movie-header-content-expanded.jsx [hunk 1]
```diff
@@ -31,6 +40,12 @@ export default React.createClass({
                     <h6>{this.textFor('productionYear')}</h6>
                     <p>{this.textFor('shortSynopsis')}</p>
                 </div>
+                <div>
+                    <Button label='Voir la bande annonce' type='button' handleOnClick={this.openTrailerPopin} />
```
Comment (Bernardstanislas) [inline app/views/movie/detail/movie-header-content-expanded.jsx:44]: No translation here ?

File: app/views/movie/trailer.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        url: PropTypes.string
+    },
+
+    /** @inheritDoc */
+    render() {
+        const {url} = this.props;
+        return (
+            <div data-demo='trailer'>
+                {url &&
+                    <iframe src={url} allowfullscreen={true} seamless={true} width="1024" height="768" />
```
Comment (Bernardstanislas) [inline app/views/movie/trailer.jsx:16]: Is it possible to give units such as `80vw` for the size ?

Comment (TomGallon) [inline app/views/movie/trailer.jsx:16]: No, in HTML 5 spec of iframe, it is mentionned that you can only specify pixels.

File: app/views/movie/detail/movie-header-content-expanded.jsx (hunk 1 above)
Comment (TomGallon) [inline app/views/movie/detail/movie-header-content-expanded.jsx:44]: yes, i forgot to do it. thx.

File: app/views/movie/trailer.jsx (hunk 2 above)
Comment (Bernardstanislas) [inline app/views/movie/trailer.jsx:16]: Fair enough

**Pull request merged**